wrappers/python/tests/nntile_core/test_tensor_strassen.py

```
@@ -0,0 +1,113 @@
+    else:
+        shape = [shared_size, matrix_shape[0], batch]
+    traits = nntile.tensor.TensorTraits(shape, tile_shape)
+    mpi_distr = [0] * traits.grid.nelems
+    A = Tensor[dtype](traits, mpi_distr, next_tag)
+    src_A = np.array(np.random.randn(*shape), dtype=dtype, order="F")
+    next_tag = A.next_tag
+
+    if tB == nntile.notrans:
+        shape = [shared_size, matrix_shape[1], batch]
+    else:
+        shape = [matrix_shape[1], shared_size, batch]
+    traits = nntile.tensor.TensorTraits(shape, tile_shape)
+    mpi_distr = [0] * traits.grid.nelems
+    B = Tensor[dtype](traits, mpi_distr, next_tag)
+    src_B = np.array(np.random.randn(*shape), dtype=dtype, order="F")
+    next_tag = B.next_tag
+
+    shape = [matrix_shape[0], matrix_shape[1], batch]
+    traits = nntile.tensor.TensorTraits(shape, tile_shape)
+    mpi_distr = [0] * traits.grid.nelems
+    C = Tensor[dtype](traits, mpi_distr, next_tag)
+    src_C = np.array(np.random.randn(*shape), dtype=dtype, order="F")
+    dst_C = np.zeros_like(src_C, dtype=dtype, order="F")
+
+    # Set initial values of tensors
+    A.from_array(src_A)
+    B.from_array(src_B)
+    C.from_array(src_C)
+
+    C.to_array(dst_C)
+    strassen[dtype](alpha, tA, A, tB, B, beta, C, 1, 1, 0)
+    C.to_array(dst_C)
+
+    nntile.starpu.wait_for_all()
+    A.unregister()
+    B.unregister()
+    C.unregister()
+
+    # Check results
+    for i in range(batch):
+        # Get result in numpy
+        termA = src_A[:, :, i] if tA == nntile.notrans else src_A[:, :, i].T
+        termB = src_B[:, :, i] if tB == nntile.notrans else src_B[:, :, i].T
+        src_C[:, :, i] = beta * src_C[:, :, i] + alpha * (termA @ termB)
+        # Check if results are almost equal
+        if (
+            np.linalg.norm(dst_C[:, :, i] - src_C[:, :, i])
+            / np.linalg.norm(src_C[:, :, i])
+            > 1e-4
+        ):
+            return False
+    return True
+
+
+# Repeat tests for different configurations
+def tests():
```

@Muxas What are our plans for testing? We should avoid this boilerplate with `pytest` and parametrized test as follows.

```python
@pytest.mark.parametrized('dtype', [np.float32, np.float64])
@pytest.mark.parametrized('trans', [False, True])
def test_strassen(dtype, trans, ...):
    ...
```

But I'm not sure that this will be work properly since multiple test could be run and multiple runtimes can be created. It might break something. Thus we at least should add `pytest-timeout` to fail faster.